nomt/src/lib.rs

```diff
@@ -165,12 +173,9 @@ impl Nomt {
         let page_cache = PageCache::new(store.clone(), &o);
         let root = store.load_root()?;
         Ok(Self {
+            commit_pool: CommitPool::new(o.fetch_concurrency),
```

Comment:
it came to my attention that `o.traversal_concurrency` is not really used anymore

Reply:
yes, I noticed this as well. we lost it somewhere, but it was before this stack.